functions.Rmd

```diff
@@ -138,7 +138,7 @@ OTHERWISE
 In R, we write these conditional statements using the keywords **`if`** and **`else`** and the following syntax:
 
 ```r
-if(condition){
+if (condition) {
```

Comment:
I prefer no spaces between an `if` and its expression (a la Java), but will defer to style guide

Reply:
[Relevant quote](https://google.github.io/styleguide/Rguide.xml#spacing):
> Place a space before left parenthesis, except in a function call.
> GOOD: 
> `if (debug)`
> BAD: 
> `if(debug)`